src/go/.gitignore

```diff
@@ -1,4 +1,8 @@
-stress-tester
-k8s-api-test
 customer-trace-tester
+!customer-trace-tester/
+k8s-api-test
+!k8s-api-test/
```

Comment:
so we want to ignore file but keep the directory with the same name? :confused:

Reply:
Yes. The file is the binary that is the result of `make build`.

Reply:
I would add absolute path then

Reply:
or add `.gitignore` to the directory which is related to the binary

Reply:
The Makefile is in this directory and the binaries are created in this directory too. This whole setup is a bit weird to me to be honest. In my opinion, each Go program should have its own space and not a common Makefile or anything.

Reply:
I've changed the "implementation" of my intention and documented it, hopefully it's clearer now.